README.md

```diff
@@ -16,18 +16,25 @@ import '@github/auto-complete-element'
 
 ```html
 <auto-complete src="/users/search" aria-owns="users-popup">
-  <input type="text" data-autocomplete-autofocus>
```

Comment:
Nice catch on this legacy attribute. We also need to remove this in the GitHub app as we convert to `<auto-complete>` elements.